doc/source/contributing_code.txt

```diff
@@ -131,11 +131,14 @@ all issues have been resolved.
 <li>Programmers begin work on a feature or fix by branching from
 develop.
 
-<li>When branch is ready, it is submitted to code review.
+<li>When a branch is ready, it is submitted to code review.
 
 <li>When code review is complete, and the changes are approved, the PR
 is merged into the develop branch.
 
+<li>Mutliple merges into the develop branch may take place between
+test cycles. (???)
+
 <li>The develop branch is tested automatically by Jenkins.
 
 <li>The develop branch is tested periodically by CDash (every ~6
```

Comment:
Does Cdash run tests?  I thought our setup was that Jenkins (at NCAR) runs the tests and sends results to cdash.